core/src/main/scala/cats/data/OneAnd.scala

```diff
@@ -209,7 +209,7 @@ sealed abstract private[data] class OneAndLowPriority4 {
         fa.head
 
       def map[A, B](fa: OneAnd[LazyList, A])(f: A => B): OneAnd[LazyList, B] =
-        fa.map(f)
+        fa.map(f)(crossVersionInstancesForLazyList)
```

Comment:
so for a user who needs to cross compile, they either have to create this `crossVersionInstancesForLazyList` themselves or have to write version specific code for every `import cats.instances.stream._` they have in the past?

Reply:
see my comment below.